packages/zoe/src/contractSupport/durability.js

```diff
@@ -36,40 +36,38 @@ harden(makeEphemeraProvider);
 /**
  *
  */
-export const makeEphemeralStoredSubscriberProvider = () => {
-  /** @type {WeakMap<Subscriber<any>, StoredSubscriber<any>>} */
+export const makePublicTopicProvider = () => {
+  /** @type {WeakMap<Subscriber<any>, import('@agoric/notifier').PublicTopic<any>>} */
   const extant = new WeakMap();
 
   /**
-   * Provide a StoredSubscriber for the specified durable subscriber.
+   * Provide a PublicTopic for the specified durable subscriber.
+   * Caches the resolution of the promise for the storageNode's path.
```

Comment:
'Caches' in what sense? Where is there data that might be dropped under some conditions?

Reply:
> 'Caches' in what sense?
Caches in the sense that if the PublicTopic has been resolved, it never need be resolved again. (No remote call to getPath())

> Where is there data that might be dropped under some conditions?

The set of PublicTopics will be lost on vat termination.

WDYT of,
```suggestion
   * Memoizes the resolution of the promise for the storageNode's path, for the lifetime of the vat.
```

Reply:
That makes more sense to me.